content/events/online-hackfest/2020-uiux/index.adoc

```diff
@@ -0,0 +1,131 @@
+---
+layout: project
+title: "Jenkins UI/UX Hackfest"
+sig: "advocacy-and-outreach"
+tags:
+  - outreach-programs
+  - events
+  - hackathon
+  - community
+  - ux
+links:
+  gitter: jenkinsci/hackfest
+opengraph:
+  image: /images/post-images/jenkins-is-the-way/jenkins-is-the-way-hackfest-opengraph.png
+---
+
+UI/UX Online Hackfest is an online event, it will take place on May 25-29.
+The goal to drive improvements in the Jenkins user experience (UX), including but not limited to user interface and user documentation. 
+The event will have the link:/blog/2020/04/30/jenkins-is-the-way/[Jenkins is the Way] theme.
+
+image:/images/post-images/jenkins-is-the-way/jenkins-is-the-way.png[Jenkins Is The Way, role=center, float=right, height=284]
+
+== What is Hackfest?
+
+This hackfest is NOT a _hackathon_.
+We do not expect participants to dedicate all their time during the event timeframe.
+Instead of that, we organize it as a hop-in/hop-off event where everybody can spend as much time as they are willing to dedicate.
+Spending a few days or a few hours is fine, any contributions matter regardless of their size.
+Jenkins experience is not required, we have newcomer-friendly stories.
+See the link:/participate/[Contribute and Participate] page for more information about how to contribute.
+
+During this hackfest, we will be organizing some online meetups and ad-hoc training sessions in different timezones.
+All these sessions will be recorded and shared shortly on our YouTube channel.
+There are no mandatory events  you must attend remotely, you are welcome to join the events you are interested in.
+At the end of the event we will be inviting participants to join our demo sessions or to record and upload these demos share asynchronously.
+
+Most active contributors will get special edition swag and other prizes, see below!
+
+== Registration
+
+NOTE: Registration form will be published soon!
+Please check this page later.
+
+== Tracks and Project ideas
+
+WARNING: This section is under development.
+More details will be added soon.
+
+=== User interface
+
+Newcomer-friendly areas:
+
+* Jenkins Look & Feel updates
+** Jira query: coming soon
+* Jenkins accessibility
+** Jira query: coming soon
+* Other newcomer-friendly issues related to user experience
+** Jira query: coming soon
+
+Advanced projects:
+
+* JEP-224: System Read permission. Improve user-experience for users of Jenkins Configuration-as-Code (jep:224[])
+** EPIC in Jenkins Jira: jira:JENKINS-12548[]
+** link:https://github.com/orgs/jenkinsci/projects/4[GitHub project] with more issues for grabs: 
+* Modification of Jenkins Plugin manager pages
+* TBD: Credentials Plugin UI improvements
+* TBD: Git Plugin 
```

Comment:
Advanced because pull requests require that the contributor has a development environment and understands where to place the desired online help.
```suggestion
* Git Plugin documentation improvements
```

Reply:
It is in the UI section, not in the docs one